settings.gradle

```diff
@@ -1 +1,3 @@
-include ':AndroidStealth', ':libraries:FileChooser', ':libraries:droidmorphing-lib', ':libraries:DirectoryChooser'
+include ':AndroidStealth', ':libraries:FileChooser', ':libraries:DirectoryChooser', ':droidmorphing-lib'
+project(':droidmorphing-lib').projectDir = new File('libraries/droid-morph/droidmorphing-lib')
```

Comment:
What did not work was `include ':libraries:droid-morph:droidmorphing-lib'`. In this case, it would try to make a project from every intermediate folder, thereby including `droid-morph` as a project (but it wasn't buildable)

What it does now, is define a project `droidmorphing-lib` and actually point gradle to the folder. Seems logical now, but it was a bitch to find out.